core/jazz_ui/src/app/secondary-components/advanced-filters/internal/advanced-filters.component.ts

```diff
@@ -85,6 +85,16 @@ export class AdvancedFiltersComponent implements OnInit {
         this.periodList=event;
         this.periodSelected=this.periodList[0];
     }
+    setSlider(event){
```

Comment:
Same as above. You can remove this method.

Reply:
i removed it